app/decorators/reviewable.rb

```diff
@@ -5,6 +5,15 @@ def class
     __getobj__.class
   end
 
+  def attributes=(attributes)
```

Comment:
Would it make sense to also set `self.public = true` when we check `attributes["reviewed"] == "1"` and `self.public = false` when `"attributes["reviewed"] == "0"`? I think this would enable the item to show up in the search results if it is checked as reviewed or hide it if it's sent back to the reviewer queue.